### ISSUE TITLE
 [Failing test][sig-node] node-kubelet-master-kubetest2

### ISSUE THREAD

**Issue by rajula96reddy** [kubernetes/kubernetes#106478]
### Which jobs are failing?

- node-kubelet-master-kubetest2

### Which tests are failing?

- kubetest2.Test

### Since when has it been failing?

2021-11-17 06:17 CET

### Testgrid link

https://testgrid.k8s.io/sig-release-master-blocking#node-kubelet-master-kubetest2

### Reason for failure (if possible)

From build log:
```
 Activated service account credentials for: [prow-build@k8s-infra-prow-build.iam.gserviceaccount.com]
+ WRAPPED_COMMAND_PID=30
+ wait 30
+ kubetest2 noop --test=node -- --repo-root=. --gcp-zone=us-west1-b --parallelism=8 '--focus-regex=\[NodeConformance\]' '--skip-regex=\[Flaky\]|\[Serial\]' '--test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"' --image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/image-config.yaml
I1117 08:20:50.245905      37 app.go:59] RunDir for this run: "/logs/artifacts/285097d1-477f-11ec-b006-ce2da5ccbb5b"
I1117 08:20:50.258122      37 app.go:94] ID for this run: "285097d1-477f-11ec-b006-ce2da5ccbb5b"
make: *** No rule to make target 'test-e2e-node'.  Stop.
F1117 08:20:50.332099      48 node.go:262] failed to run ginkgo tester: exit status 2
Error: exit status 255
+ EXIT_VALUE=1
+ set +o xtrace 
```

### Anything else we need to know?

This needs to be resolved for 1.23.0-beta.0 release scheduled on 17/11/2021

### Relevant SIG(s)

/sig node

**Comment by aojea** [comment]
this job was added 5 hours ago and it seems it never worked, so it should not be blocking but we should be careful adding jobs to the blocking tab to avoid false positives

https://github.com/kubernetes/test-infra/pull/24308

/assign @amwat @SergeyKanzhelev @Namanl2001

**Comment by Namanl2001** [comment]
The failure seems to be kind of new and might need a few iterations to fix it. But as this needs to be resolved for 1.23.0-beta.0 release scheduled on 17/11/2021, we can move this job from blocking tabs to a non-blocking one, may be to `sig-node-kubelet` ?

**Comment by palnabarun** [comment]
+1 on moving this job to informing boards instead of blocking. I'm not sure if this job should even be a `release-informing` job given that it is pretty new and tracks the sanity of migration from kubetest to kubetest2.

What do you think?

**Comment by palnabarun** [comment]
Refer to the Slack thread on context behind the above PR ☝🏽 

https://kubernetes.slack.com/archives/C0BP8PW9G/p1637142313194000

**Comment by dims** [comment]
@Verolop once it goes green please (to add it to informing, right now it's broken and @Namanl2001 is working on it)

**Comment by SergeyKanzhelev** [comment]
/triage accepted
/priority important-soon
/kind cleanup
/remove-kind 

Fixing issue priority. New link to the test: https://testgrid.k8s.io/sig-node-release-blocking#node-kubelet-master-kubetest2

**Comment by amwat** [comment]
Ahh yes, it was always intended to be introduced as non-blocking. Thanks for catching this

**Comment by Namanl2001** [comment]
This issue can be closed now 😄  ref: https://github.com/kubernetes/test-infra/pull/24409#issuecomment-976175083

**Comment by aojea** [comment]
> This issue can be closed now 😄 ref: [kubernetes/test-infra#24409 (comment)](https://github.com/kubernetes/test-infra/pull/24409#issuecomment-976175083)

Awesome, great job
/close
